admin-manual/installation-setup/customization/dashboard-config.rst

```diff
@@ -721,6 +721,59 @@ user, you can change it via the command-line:
        /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/python manage.py changepassword <username>
    ";
 
+CLI configuration pipeline and registration on the Storage Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On new installations, the superuser can be configured and the pipeline
+registered on the Storage Service using the command-line instead of the GUI.
+These commands configure the pipeline and register it on the storage service
+exactly like the GUI for the initial configuration screen shown after the
+installation would:
+
+.. code:: bash
+
+   sudo -u archivematica bash -c " \
+       set -a -e -x
+       source /etc/default/archivematica-dashboard || \
+           source /etc/sysconfig/archivematica-dashboard \
+               || (echo 'Environment file not found'; exit 1)
+       cd /usr/share/archivematica/dashboard
+       /usr/share/python/archivematica-dashboard/bin/python manage.py install \
+           --username=<username> \
+           --password=<password> \
+           --email=<email-address> \
+           --org-name=<org-name> \
+           --org-id=<org-id> \
+           --api-key=<api-key>\
+           --ss-url=<ss-ulr> \
+           --ss-user=<ss-username> \
+           --ss-api-key=<ss-api-key> \
+           --whitelist=<whitelist>
+   ";
+
+Where:
```

Comment:
In the list below, can you use [inline literals](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup) for things like `api-key`, `test` etc?

Also the indentation in the enumeration is not needed, the fact that is there causes the HTML to be rendered inside a `blockquote`, see:

![image](https://user-images.githubusercontent.com/606459/47310121-0b650200-d5eb-11e8-88cf-ca546861ef46.png)